src/components/reference-book/page.cjsx

```diff
@@ -13,7 +13,7 @@ GetPositionMixin = require '../get-position-mixin'
 {ReferenceBookStore} = require '../../flux/reference-book'
 {ReferenceBookExerciseStore} = require '../../flux/reference-book-exercise'
 
-EXERCISE_MATCHER = new RegExp('#ost\/api\/ex\/(.*)')
+EXERCISE_MATCHER = new RegExp('\/exercises\/(.*)')
```

Comment:
ick, I wonder if we need/should match based on a CSS selector like `[data-type='exercise'] > [data-type='problem'] .os-embed` instead